ironfish/src/rpc/clients/client.ts

```diff
@@ -821,6 +823,15 @@ export abstract class RpcClient {
     },
   }
   multisig = {
+    aggregateSignatureShares: (
```

Comment:
We should either rename this to match `signFrost` or rename `signFrost` to match this.

Reply:
Merging as is for now, we'll settle on naming in a followup